modules/microshift-rhoai-query-model-con.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-rhoai-query-model-con_{context}"]
+= About querying your AI model
+
+Querying your model through the API is also called model inferencing. Model inferencing is most often used to retrieve information, automate tasks, make predictions, provide data insights, or perform actions.
+
+In general, queries must be constructed using a format compatible with the AI model being used. A model-serving runtime completes this process automatically. The model processes the query according to the underlying training and data, then provides an output. The output is expected to align with the purpose of the model itself, whether that be to give an answer, make a prediction, or perform a task.
```

Comment:
> A model-serving runtime completes this process automatically.

What process? Query construction? Or that it processes the requests?

Reply:
My understanding is that the MSR formats the query so it can be digested by the model. Clarified (unless that isn't accurate).